### PR TITLE
Add designathon promo on the home page

### PR DESCRIPTION
**Pull request by GBKS**
Without this promo there's no way to discover the event from the website. I put this together fairly quickly for the announcement today, feedback appreciated.

[Preview](https://deploy-preview-924--bitcoin-design-site.netlify.app)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -22,6 +22,22 @@ Everyone is welcome to join and [participate]({{ '/contribute' | relative_url }}
...
+   image = "/assets/images/home/designathon.png"
+   retina = "/assets/images/home/designathon@2x.png"
+   mobile = "/assets/images/home/designathon-mobile.png"
+   mobileRetina = "/assets/images/home/designathon-mobile@2x.png"
+   alt-text = "Bitcoin Designathon promo banner"
+   width = 800
+   height = 367
+%}
+
+We are organizing a designathon. Imagine a traditional hackathon, but this time focused only on design. Our goal is to motivate designers and builders to get together and collaboratively explore ideas and concepts in and around the bitcoin ecosystem. Come join us.
```
Comment (sbddesign) [inline index.md:37]:
```suggestion
We are organizing a designathon! Imagine a traditional hackathon, but this time focused only on design. Our goal is to motivate designers and builders to get together and collaboratively explore ideas and concepts in and around the bitcoin ecosystem. Come join us!
```

**Review by sbddesign — APPROVED** [review]
Looks nice! Left a suggestion to make the paragraph perkier with exclamation marks, but don't consider that a blocking change.

Also would be nice:
- if the event was higher above the fold (but this is time sensitive and that would take a little work to make that look right)
- if the image could be a link (but requires venturing out from the traditional jekyll includes we use)

So having said all that, I think we should get it on the site and start promoting it. Can tweak it later.

File: index.md (hunk 1 above)
Comment (Bosch-0) [reply index.md:37]: Agree this should be ATF, it's given secondary importance being below it. Lets get it out but

**Review by Bosch-0 — DISMISSED** [review]

**Comment by Bosch-0** [comment]
We could add the persistent banner stickied to the top of the page. I know you didn't like this approach Christoph but It looks fine and brings it ATF without having to move too many things around moving too many parts.

**Comment by GBKS** [comment]
Thanks for the feedback. I just accepted the copy change and updated the image to be a link (using HTML instead of markdown - I think it's OK because this is a temporary short-term thing).

For a persistent banner, please also consider that we have a sticky nav bar, and an expanding menu on mobile. If that banner is also sticks and sits above the nav bar, the code for the menu needs to be updated to make that positioning work, and for the expanding menu to account for the height difference. It takes some CSS and JS fiddling to get this right.

Why not just take over the top banner altogether?
image

Only downside is that it wouldn't be a direct link, people would have to scroll down a bit to get to that. Not a huge issue though.

**Review by danielnordh — APPROVED** [review]
LGTM

**Comment by GBKS** [comment]
Merged this one, as the bolt.fun announcement will happen in the next hour (fingers crossed) and there are some links floating around that lead to our home page, not the event page.

**Pull request merged**